htdocs/js/notebook/notebook_model.js

```diff
@@ -211,14 +211,26 @@ Notebook.create_model = function()
             RCloud.UI.selection_bar.update(this.cells);
         },
         hide_selected_cells_results: function() {
-            _.each(this.get_selected_cells(), function(cell) {
-                cell.hide_cell_result();
-            });
+            if(this.get_selected_cells()) {
```

Comment:
I know it's annoying to deal with little nit-picky issues, but please test even the most trivial code.

`this.get_selected_cells()` always returns an array, which is truthy...